stable/rabbitmq/values-production.yaml

```diff
@@ -96,12 +96,13 @@ service:
   ## Node port
   ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
   ##
-  nodePort: 5672
+  # nodePort: 30672
 
   ## Amqp port
   ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
   ##
-  amqpPort: 5672
+  port: 5672
```

Comment:
Undo this change too.

Reply:
Isn't ``amqpPort`` a typo?

I cannot see that reference inside the ``svc-headless.yaml`` nor the ``svc.yaml`` templates.
Also, commenting out the ``port`` value, will leave tjhe port value null:

https://github.com/helm/charts/blob/master/stable/rabbitmq/templates/svc-headless.yaml#L17
https://github.com/helm/charts/blob/master/stable/rabbitmq/templates/svc.yaml#L21

Reply:
Yes... I think it's a bug introduced on this commit: https://github.com/helm/charts/commit/8dc228412f2e63877bacb0bc5502855aaa87bcbc
Could you update the README.md so it uses `port` instead of `amqpPort`?